attributes/default.rb

```diff
@@ -1,22 +1,27 @@
 #<> Gerrit version
-default['gerrit']['version'] = "2.12.7"
+default['gerrit']['version'] = "2.14.4"
```

Comment:
For the upgrade process we need multiple steps, there are some tasks to be done on the current version of the production server (which is 2.12.7).

The version should either be 2.12.7 like production or 2.14.5.1 like the current stable release (which we need as interim step during the upgrade) or 2.15-rc2 like the currently planned fully upgraded version

Reply:
then 2.14.5.1 makes most sense to me